Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -509,6 +509,7 @@
 workflows[140.116] = ['',['RunCommissioning2022B','HLTDR3_2022','RECONANORUN3_reHLT','SKIMCOMMISSIONINGRUN3_reHLT']]
 workflows[140.117] = ['',['RunCosmics2022B','HLTDR3_2022','RECOCOSMRUN3_reHLT','SKIMCOSMICSRUN3_reHLT']]
 #workflows[140.118] = ['',['RunParkingBPH2022B','HLTDR3_2022','RECONANORUN3_reHLT','SKIMPARKINGBPHRUN3_reHLT']]
+workflows[140.119] = ['',['RunScoutingPFRun32022D', 'NanoRun3ScoutingPF']]
```

Comment:
can you please move the definition of this workflow in relval_nano.py ; possibly with the definition of the steps too